test/unit/tracks/text-track-controls.test.js

```diff
@@ -435,7 +436,11 @@ test('chapters should be displayed when remote track added and load event fired'
 
   equal(chaptersEl.track.cues.length, 2);
 
-  TestHelpers.triggerDomEvent(chaptersEl, 'load');
+  if (player.tech_.featuresNativeTextTracks) {
+    TestHelpers.triggerDomEvent(chaptersEl, 'load');
```

Comment:
I decided that a general solution would be better than specifically checking for IE8. However, I'll add a comment about this.